### DISCUSSION TITLE
Contribute to dailydotdev

### DISCUSSION THREAD

**Discussion by sxryadipta** [dailydotdev/daily#1314] [General]
I want to contribute to this project. How to get started?

**Comment by sshanzel** [comment]
Hi! I appreciate your interest, feel free to check our issues page about items with labels of [help-wanted](https://github.com/dailydotdev/daily/issues?q=is%3Aissue+is%3Aopen+label%3A%22help+wanted%22). If you find something that you like to work on, and still available, just send a comment on the actual issue. From there, you can use our GitPod setup to do some local development.
